src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -173,6 +173,21 @@ public Category GetCategory
         }
 
         /// <summary>
+        /// Get Element Type
+        /// </summary>
+        public Element ElementType
+        {
+            get
+            {
+                // Get the active Document
+                Autodesk.Revit.DB.Document document = DocumentManager.Instance.CurrentDBDocument;
+
+                return document.GetElement(this.InternalElement.GetTypeId()).ToDSType(true);
```

Comment:
How do you want to do the `FamilyType` return? You can't cast `Element` to `FamilyType` and `Element` is what `GetElement()` returns. You also can't use the static method `FamilyType.FromExisting()` because again, we have an `Element` not the `FamilyType`. `ToDSType()` actually returns a `FamilyType` because it uses a dynamic variable. Should we implement that here?